coordinator/tests/services/translation/response/release/release-medication-request.spec.ts

```diff
@@ -445,8 +426,38 @@ describe("dispenseRequest", () => {
     })
   })
 
+  test("handles continuous dispensing by adding numberOfRepeatsAllowed", () => {
```

Comment:
Might be clearer to call it "repeat prescribing" in the test name. Should probably add a test for repeat dispensing too